mitty/simulation/genome/sampledgenome.py

```diff
@@ -1 +1,31 @@
-""""""
\ No newline at end of file
+import pysam
+from numpy.random import RandomState
+
+
+def assign_random_gt(input_vcf, outname, sample_name="HG", default_af=0.01, seed=None):
```

Comment:
instead of a file name, please use a file pointer, this makes the function much more flexible, please see `simulate-variants`